tests/bluetooth/controller/ctrl_min_used_chans/src/main.c

```diff
@@ -0,0 +1,152 @@
+#include "ull_tx_queue.h"
+#include "ull_llcp.h"
+#include "ull_llcp_internal.h"
+
+#include "helper_pdu.h"
+#include "helper_util.h"
+
+struct ull_cp_conn conn;
+
+static void setup(void)
+{
+	test_setup(&conn);
+}
+
+
+
+/* +-----+                     +-------+                  +-----+
+ * | UT  |                     | LL_A  |                  | LT  |
+ * +-----+                     +-------+                  +-----+
+ *    |                            |                         |
+ *    | Start                      |                         |
+ *    | Min used chans Proc.       |                         |
+ *    |--------------------------->|                         |
+ *    |                            |                         |
+ *    |                            | LL_MIN_USED_CHANS_IND   |
+ *    |                            |------------------------>|
+ *    |                            |                 'll_ack'|
+ *    |                            |                         |
+ *    |                            |                         |
+ */
+void test_min_used_chans_sla_loc(void)
+{
+	u8_t err;
+	struct node_tx *tx;
+
+	struct pdu_data_llctrl_min_used_chans_ind local_muc_ind = {
+		.phys = 1,
+		.min_used_chans = 2
+	};
+
+	struct pdu_data_llctrl_min_used_chans_ind remote_muc_ind = {
+		.phys = 1,
+		.min_used_chans = 2
+	};
+
+	/* Role */
+	test_set_role(&conn, BT_HCI_ROLE_SLAVE);
+
+	/* Connect */
+	ull_cp_state_set(&conn, ULL_CP_CONNECTED);
+
+	/* Initiate an LE Ping Procedure */
+	err = ull_cp_min_used_chans(&conn, 1, 2);
+	zassert_equal(err, BT_HCI_ERR_SUCCESS, NULL);
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Tx Queue should have one LL Control PDU */
+	lt_rx(LL_MIN_USED_CHANS_IND, &conn, &tx, &local_muc_ind);
+	lt_rx_q_is_empty(&conn);
+
+	/* Rx */
+	lt_tx(LL_MIN_USED_CHANS_IND, &conn, &remote_muc_ind);
+
+	/* TX Ack */
+	event_tx_ack(&conn, tx);
+
+    /* Done */
+	event_done(&conn);
+
+	/* Release tx node */
+	ull_cp_release_tx(tx);
+
+	/* There should not be a host notifications */
+	ut_rx_q_is_empty();
+}
+
+void test_min_used_chans_mas_rem(void)
+{
+	struct pdu_data_llctrl_min_used_chans_ind remote_muc_ind = {
+		.phys = 1,
+		.min_used_chans = 2
+	};
+
+	/* Role */
+	test_set_role(&conn, BT_HCI_ROLE_MASTER);
+
+	/* Connect */
+	ull_cp_state_set(&conn, ULL_CP_CONNECTED);
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Rx */
+	lt_tx(LL_MIN_USED_CHANS_IND, &conn, &remote_muc_ind);
+
+	/* Done */
+	event_done(&conn);
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Tx Queue should have no LL Control PDU */
+	lt_rx_q_is_empty(&conn);
+
+	/* Done */
+	event_done(&conn);
+
+	/* There should not be a host notifications */
+	ut_rx_q_is_empty();
+
+}
+
+void test_main(void)
+{
+	ztest_test_suite(ping,
+			ztest_unit_test_setup_teardown(test_min_used_chans_sla_loc, setup, unit_test_noop),
+			ztest_unit_test_setup_teardown(test_min_used_chans_mas_rem, setup, unit_test_noop)
+			);
+
+	ztest_run_test_suite(ping);
```

Comment:
Are we missing a unittest for the case where you are master or slave, and receive a min_used_chans request?